src/renderers/webvr/WebVRManager.js

```diff
@@ -104,40 +108,49 @@ function WebVRManager( renderer ) {
 
 		//
 
-		var pose = frameData.pose;
-		var poseObject = poseTarget !== null ? poseTarget : camera;
+		var stageParameters = device.stageParameters;
 
-		if ( pose.position !== null ) {
+		if ( stageParameters ) {
 
-			poseObject.position.fromArray( pose.position );
+			standingMatrix.fromArray( stageParameters.sittingToStandingTransform );
 
 		} else {
 
-			poseObject.position.set( 0, 0, 0 );
+			standingMatrix.makeTranslation( 0, scope.userHeight, 0 );
 
 		}
 
+
+		var pose = frameData.pose;
+		var poseObject = poseTarget !== null ? poseTarget : camera;
+
+		// We want to manipulate poseObject by its position and quaternion components since users may rely on them.
+		poseObject.matrix.copy( standingMatrix );
+		poseObject.matrix.decompose(poseObject.position, poseObject.quaternion, poseObject.scale);
+
 		if ( pose.orientation !== null ) {
 
-			poseObject.quaternion.fromArray( pose.orientation );
+			tempQuaternion.fromArray ( pose.orientation );
+			poseObject.quaternion.multiply( tempQuaternion );
 
 		}
 
-		var stageParameters = device.stageParameters;
-
-		if ( stageParameters ) {
+		if ( pose.position !== null ) {
 
-			standingMatrix.fromArray( stageParameters.sittingToStandingTransform );
+			tempQuaternion.setFromRotationMatrix(standingMatrix);
+			tempPosition.fromArray( pose.position );
+			tempPosition.applyQuaternion(tempQuaternion);
+			poseObject.position.add( tempPosition );
 
 		} else {
 
-			standingMatrix.makeTranslation( 0, scope.userHeight, 0 );
```

Comment:
Not using userHeight anymore is intentional?

Reply:
The diff is misleading. This `if` block was moved up to line 113

Reply:
I see the standingMatrix and scope.userHeight is accounted for above. If position is null I think this else branch would reset the height back to 0 overriding the scope.userHeight, doesn’t it?

Reply:
Ah, you're right. Good catch.

Reply:
Removing the else would do I think